rockcraft.yaml

```diff
@@ -65,3 +65,17 @@ parts:
       umount --recursive $CRAFT_OVERLAY/dev
       umount $CRAFT_OVERLAY/proc
       umount $CRAFT_OVERLAY/etc/resolv.conf
+  prometheus-exporter:
+    plugin: python
+    stage-packages:
+      - python3-venv
+    source: https://github.com/pyToshka/wazuh-prometheus-exporter.git
+    source-depth: 1
+    source-type: git
+    source-commit: 7ab31b3f73b0ce344bd7e4d0deb1d544ae1b7d1c
+    organize:
+      main.py: prometheus_exporter.py
+    prime:
```

Comment:
I think there are other files we want to remove from the repo no ?
Isn't it easier to reverse the logic and only copy what we need ?